### ISSUE TITLE
Use Zabbix API token instead user login

### ISSUE THREAD

**Issue by thiagotkt** [grafana/grafana-zabbix#1513]
There are too many Grafana user that use Zabbix plugin that generate many sessions that constantly updating lastaccess on table sessions that is causing waits on database. The new feature, API token, not use session table then it will decrease updates on the table and can be helpful to large environment.

**Comment by Th3Tubby3d** [comment]
`{ "jsonrpc" : "2.0", "method" : "host.get", "params" : {}, "id" : 1, "auth" : "$authToken" }` 

If the ask is to be able to manipulate the auth field directly without any of the other authentication mechanisms ("$authToken") in my generic example of a Zabbix API call, I would similarly appreciate this functionality.

**Comment by thiagotkt** [comment]
Yes, I would like to use the api token instead of the user and password in the datasource configuration. Zabbix doc ref: https://www.zabbix.com/documentation/6.0/en/manual/web_interface/frontend_sections/administration/general#api-tokens

**Comment by X5-Evgeny-Semerikov** [comment]
This is nice feature if it will be implemented

**Comment by jgarconi** [comment]
I'm having the same issue. Waiting for updates.

**Comment by Raimond56** [comment]
Upvote, would be nice to have API Token audentication.
